src/interactive/results.ts

```diff
@@ -422,10 +426,10 @@ function isResultInLineRange(editor: vscode.TextEditor, result: Result, range: v
 
 // goto frame utilties
 
-export async function openFile(path: string, line: number = undefined) {
-    line = line || 1
-    const start = new vscode.Position(line - 1, 0)
-    const end = new vscode.Position(line - 1, 0)
+export async function openFile(path: string, line: number | undefined = undefined) {
+    const newLine = line ?? 1
```

Comment:
Pretty sure `||` is more correct here, no? The result should always be at least 1, but `0 ?? 1` would return 0, right?

Reply:
Yep, didn't think of that!